python/py-mpd2/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-mpd2
+python.rootname     python-mpd2
+version             3.0.4
+revision            0
+
+categories-append   devel audio
+platforms           darwin
+supported_archs     noarch
+license             LGPL-3
```

Comment:
this should be `LGPL-3+` based on what upstream states: "python-mpd2 is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version."